karma.conf.js

```diff
@@ -89,7 +89,7 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
```

Comment:
@misscs Yeah, I agree for sure. I switched this to Chrome because PhantomJS didn't seem to respect the viewport options that I set. Without the viewport settings, scrolling doesn't actually occur because it automatically makes the viewport large enough to fit all the page content.